app_classes/amity.py

```diff
@@ -44,14 +66,22 @@ def add_person(self, name, role, wants_accomodation):
 			print('Another user exists with the same name')
 			return 'Another user exists with the same name'
 		else:
+			random_office = None
+			random_living_space = None
+			self.func_office_available()
+			self.func_available_living_space()
 			if role == 'STAFF':
 				if wants_accomodation == 'N':
```

Comment:
You needn't check wants_accommodation for a staff since you know a staff member definitely doesn't need accommodation